src/app/lib/analyticsUtils/index.js

```diff
@@ -188,14 +188,20 @@ export const getAtUserId = () => {
   // Attempting to track these users is just tracking that proxy, causing all opera mini visitors to have the same id
   if (isOperaProxy()) return null;
 
+  let val;
   const cookieName = 'atuserid';
-  const cookie = Cookie.getJSON(cookieName);
-  let val = pathOr(null, ['val'], cookie);
-  const expires = 397; // expires in 13 months
 
-  if (!cookie || !val) {
+  try {
+    const cookie = JSON.parse(Cookie.get(cookieName));
+    val = pathOr(null, ['val'], cookie);
+
+    if (!cookie || !val) {
+      throw new Error();
+    }
+  } catch (error) {
```

Comment:
Doesn't seem very readable to use a catch block to facilitate a 'non-error' code path? Am I right in saying if the user doesn't have the cookie the code throws on line 199 and the val is set to uuid? 

Is the try catch intended to catch a JSON parse error? If so it might be better to wrap the `JSON.parse` call only and set val to `uuid`. I'd also log out the error so we can look into any bugs around the JSON failing to parse.

Reply:
Thanks for the suggestion. Have done this now.